tests/_test_process_executor.py

```diff
@@ -203,6 +208,10 @@ def test_processes_terminate_on_executor_gc(self):
         # reference when we deleted self.executor.
         t_deadline = time.time() + 1
         while executor_reference() is not None and time.time() < t_deadline:
+            if IS_PYPY:
+                # XXX: this is only required under PyPy for some unknown
+                # reason:
+                gc.collect()
```

Comment:
There is some information about this in http://doc.pypy.org/en/latest/cpython_differences.html#differences-related-to-garbage-collection-strategies

Reply:
In particular,

>  It also means that weak references may stay alive for a bit longer than expected

Reply:
Thanks, indeed I read than I can update the comment, now.